Doppler.Jobs.Server/appsettings.json

```diff
@@ -1,12 +1,4 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information",
-      "Hangfire": "Warning"
-    }
-  },
```

Comment:
I like this declarative configuration in appsettings in place of code. Is it really necessary to use the code one?

Reply:
no, we can use this configuration for each transformation, I'll revert this change

Reply:
Done in this commit https://github.com/MakingSense/doppler-jobs/pull/3/commits/884e90723c61163e11c5fd7433c4f2431195d09f